src/version.c

```diff
@@ -28,3 +29,22 @@ const char *const BUILD_VERSION = GIT_DESCRIPTION;
 #else
 const char *const BUILD_VERSION = VERSION;
 #endif
+
+static int current_year(void) {
+	const time_t t = time(NULL);
+	struct tm tm = *localtime(&t);
+	return tm.tm_year + 1900;
+}
+
+extern char *copyright_banner(void) {
+	char *msg = NULL;
+	xasprintf(&msg,
+	          "Copyright (C) 1998-%d Ivo Timmermans, Guus Sliepen and others.\n"
```

Comment:
No, this is not how copyright notices are supposed to work. This should be reverted.

A better way would be to have some script analyzing the git log and checking against copyright messages in the source code. In particular, copyright headers at the top of source files should have the year of the last modification of that source file, and the copyright banner we print should have the year of the last modification to anything in src/. But it's better to do this in a different pull request.